debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/spi/SlotState.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql.spi;
+
+/**
+ * A simple data container that holds the state of the current slot
+ */
+public class SlotState {
+    private final Long latestFlushedLSN;
```

Comment:
Minor nit: I think we used `Lsn` casing in other places (applies in some other parts of the PR, too).

Reply:
I looked around, and both casings seems to be used. For example, all local variables in SlotState use this casing (`LSN`) as well as most of `PostgresConnection`.

I am happy to standardize on one of the other, but (AFAICT) it seems like most local variables use the all upcase version and methods use the `Lsn` version. Let me know which you prefer

Reply:
Let's use `Lsn` consistently. It's the style we generally prefer (as you say, there still may be existing usages of the other one).